### PR TITLE
Create superbuild for sctpd

### PR DESCRIPTION
**Pull request by amarpad**
## Summary

Create superbuild for sctpd

## Test Plan

s1aptester

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by rdefosse** [comment]
@amarpad 

The OAI pipeline run `https://jenkins-oai.eurecom.fr/job/MAGMA-MME-production/3426/` is a false positive.

A GitHub service was down and as a result, the OAI pipeline was in bypass mode.

Please re-trigger it again.
Sorry for the inconvenience
Raphael

File: lte/gateway/Makefile [hunk 1]
```diff
@@ -178,7 +178,7 @@ clang_warning_oai_upload: build_oai_clang ## Generate and then upload warning st
 		--data-urlencode "entry.599425517=$(shell grep '\[-Wtypedef-redefinition\]' /tmp/clang-build.oai.log | wc -l)" \
 		https://docs.google.com/forms/d/e/1FAIpQLScKB3nLPASxzr4AXW5_yeHCjkEURY0K9OAFPIyNFzkA5CY_kw/formResponse?usp=pp_url
 
-build_sctpd: format_sctpd build_common ## Build SCTPD
+build_sctpd: format_sctpd ## Build SCTPD
```
Comment (themarwhal) [inline lte/gateway/Makefile]: We probably don't need the comment here anymore?

**Review by themarwhal — APPROVED** [review]
lgtm!

File: lte/gateway/Makefile (hunk 1 above)
Comment (amarpad) [reply lte/gateway/Makefile]: Good catch thanks!

**Review by rdefosse — APPROVED** [review]

**Review by ardzoht — APPROVED** [review]
This is great, lgtm, thanks!

**Pull request merged**